src/Form/ColumnsExtractor.php

```diff
@@ -147,4 +150,20 @@ public function getCustomFieldsNumberOfColumns()
     {
         return $this->customFieldsNumberOfColumns;
     }
+
+    /**
+     * Gets the config for form, formbuilder and similar classes.
+     *
+     * @param FormInterface|FormConfigInterface $form
+     *
+     * @return FormConfigInterface
+     */
+    public static function getConfigOf($form)
```

Comment:
Simon, can you explain if there is any other idea behind it rather than changing "$formElement->getConfig()" to "$this->getConfigOf($formElement)"?

Reply:
Now ColumnsExtractor supports not only Form, but also FormBuilder

see https://github.com/EmchBerger/cube-common-bundle/pull/115#discussion_r239458164

Reply:
Ok, I see the point. But if it is static, maybe it would be better to use `self::getConfigOf($formElement)` instead of `$this->getConfigOf($formElement)` ?

Reply:
done in a945f06

Reply:
Ok, static even better then self.